### PR TITLE
[MAT-6046] Update VSAC Connection to use UMLS API KEY in Requests - Bonnie Proper

### PR DESCRIPTION
**Pull request by jkotanchik-SB**
VSAC is dropping support for its TGT/ST based authn flow. Instead, VSAC SVS API will support Basic Authentication in requests with the password set to the requester's UMLS API KEY.

- Removed methods related to generating TGT/STs.
- Removed workflows around checking for authorization based on the state of the TGT.
- Updated method params to expect an api key in place of the TGT/ST.
- Added Basic Authn headers to VSAC SVS requests.

**Submitter:**
- [x] This pull request describes why these changes were made.
- [x] Internal ticket for this PR: [MAT-6046](https://jira.cms.gov/browse/MAT-6046)
- [ ] Internal ticket links to this PR
- [x] Code diff has been done and been reviewed
- [x] Tests are included and test edge cases
- [x] Tests have been run locally and pass

**Reviewer 1:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

**Reviewer 2:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

### CODE REVIEW

File: lib/util/vsac_api.rb [hunk 1]
```diff
@@ -204,8 +183,7 @@ def get_valueset(oid, options = {})
       # hash = {vs_vsac_options: ___, value_set: {oid: ___} }
       #
       def get_multiple_valuesets(needed_value_sets)
-        raise VSACNoCredentialsError.new unless @ticket_granting_ticket
-        raise VSACTicketExpiredError.new if Time.now > @ticket_granting_ticket[:expires]
```
Comment (adongare) [inline lib/util/vsac_api.rb:208]: class `VSACTicketExpiredError` could be removed if not being used anymore?

**Review by adongare — APPROVED** [review]
